filters/regional/filter_54_AWAvenue-Adblock-RuleRule/configuration.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "秋风广告规则",

```

Comment:
The name and description must be in English here.

Reply:
It has been modified as required, please review, thank you!